.circleci/config.yml

```diff
@@ -0,0 +1,217 @@
+            );
+            curl -L -o geckodriver.tar.gz $GECKODRIVER_URL
+            gunzip -c geckodriver.tar.gz | tar xopf -
+            chmod +x geckodriver
+            sudo mv geckodriver /bin
+            geckodriver --version
+      - run:
+          name: Install Firefox
+          command: |
+            sudo npm i -g get-firefox
+            get-firefox --branch nightly --platform linux --extract --target ~/project/
+            ~/project/firefox -v
+            firefox --version
+      - run:
+          name: test
+          command: npm test
+      - run:
+          name: coverage
+          command: npm run codecov
+      - run:
+          name: integration test
+          command: npm run integration
+  package-addon-staging:
+    docker:
+      - image: circleci/node
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Package addon for staging
+          command: |
+            npm run package
+            npm run sign:stage
+      - store_artifacts:
+          path: /tmp/signed-stage.xpi
+          destination: signed-stage.xpi
+
+  package-addon-production:
+    docker:
+      - image: circleci/node
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Package addon for production
+          command: |
+            npm run package
+            npm run sign:prod
+      - store_artifacts:
+          path: ~/project/addons/signed-prod.xpi
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./*
+  publish-github-release:
+    docker:
+      - image: cibuilds/github:0.10
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: "Publish Release on GitHub"
+          command: |
+            VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[",]//g' | tr -d '[[:space:]]')
+            echo ${VERSION}
+            echo ${GITHUB_TOKEN}
+            echo ${CIRCLE_PROJECT_USERNAME}
+            echo ${CIRCLE_PROJECT_REPONAME}
+            echo ${CIRCLE_SHA1}
+            ghr -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ./project/signed-prod.xpi
```

Comment:
If I've followed everything correctly, the signed addon is located at `./project/addons/signed-prod.xpi` (rather than `./project/signed-prod.xpi`).

The error message from `ghr` is misleading to me, but I think it's happening because the path doesn't exist, therefore the caller forgot an argument, and therefore tells the caller to supply two arguments.